tracing/tests/enabled.rs

```diff
@@ -0,0 +1,30 @@
+// liballoc is required because the test subscriber cannot be constructed
+// statically
+#![cfg(feature = "alloc")]
```

Comment:
OH! this is the reason we didn't see a test failure --- there's no "alloc" feature flag on v0.1.x, because even the `no_std` version of tracing always requires liballoc. So, this is preventing the file from ever being compiled, because the feature does not exist.

We need to remove this:
```suggestion
```

Reply:
ohhhh that makes sense! So it should run on `master` right?